tests/conformance/testdata/Dockerfile.reusebase

```diff
@@ -1,4 +1,4 @@
-FROM registry.centos.org/centos/centos:centos7 AS base
+FROM docker.io/library/centos:centos7 AS base
```

Comment:
Quick note on this: [tests/registries.conf](https://github.com/containers/buildah/blob/d18a4f30bcdb992d10610f6b995b4f2a9cc27bbb/tests/registries.conf) maps this to `mirror.gcr.io`, another registry we have no control over. It might be safer to `skopeo copy --all` the desired centos images, and replace `docker.io/library` with `quay.io/libpod`. Your call.